schema/swagger.yml

```diff
@@ -563,12 +563,6 @@ paths:
           type: string
```

Comment:
Swagger version needs to be updated.

Reply:
Quick notice: `1.0.27` wasn't published. @MaciejBaj We gotta add a task on the workflow when merging a PR containing Swagger API bump.